### PR TITLE
New useModalCss hook for Modals

### LINKED ISSUE #1553: Refactor css Modal classes as a hook
In the modals React component functions, the css classes `['modal', 'fade'] are hardcoded a number of times. These could be moved into a hook that is called by the wrapped modal instead

### PR DESCRIPTION
**Pull request by jermnelson**
Fixes #1553

### CODE REVIEW

File: src/components/ResourceTemplateChoiceModal.jsx [hunk 1]
```diff
@@ -2,19 +2,17 @@
...
+import ModalWrapper, { useModalCss } from 'components/ModalWrapper'
 import PropTypes from 'prop-types'
 import { hideModal } from 'actions/index'
 
 const ResourceTemplateChoiceModal = (props) => {
   const dispatch = useDispatch()
   const show = useSelector(state => state.selectorReducer.editor.modal === 'ResourceTemplateChoiceModal')
 
-  const classes = ['modal', 'fade']
   let display = 'none'
```
Comment (justinlittman) [inline src/components/ResourceTemplateChoiceModal.jsx]: `const display = show ? 'block' : 'none'` would be more concise and clearer.

File: src/components/editor/GroupChoiceModal.jsx [hunk 2]
```diff
@@ -34,16 +34,14 @@ const GroupChoiceModal = (props) => {
     event.preventDefault()
   }
 
-  const classes = ['modal', 'fade']
   let display = 'none'
```
Comment (justinlittman) [inline src/components/editor/GroupChoiceModal.jsx]: Same as above and elsewhere.

Comment (justinlittman) [reply src/components/editor/GroupChoiceModal.jsx]: And perhaps this should be made a function in ModalCss as well?

Comment (jermnelson) [reply src/components/editor/GroupChoiceModal.jsx]: Done, using new `useDisplayStyle` hook

**Pull request merged**